src/components/component-collection/ComponentCollection.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import ComponentCollection from "./ComponentCollection";
+
+import imageFile1 from "./images/test-image-1.png";
+import imageFile2 from "./images/test-image-2.png";
+import imageFile3 from "./images/test-image-3.png";
+import imageFile4 from "./images/test-image-4.png";
```

Comment:
I would move these images to the `/assets` directory and update the path. Also, give your images semantic names, so their names describe what they are. Even something like `fall-collection-1.png`, etc. would be good. There are a lot of developers that will be testing images in this build, so we want to make sure they don't have to look through the images to figure out what they are.